src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int totalSupply = 0;
+    private int totalBuy = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        writeToFile(toFileName);
+    }
```

Comment:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
    }
```
your logic should be divided like this